crocket/src/main/java/com/crocket/view/LevelView.java

```diff
@@ -19,9 +19,7 @@ public class LevelView extends JLayeredPane implements ILevelView, IModelVisuali
     private TextureManager textureManager;
     private int levelHeight;
     private int levelWidth;
-    private PowerPanel pPanel = new PowerPanel();
-
-    Dimension screen = Toolkit.getDefaultToolkit().getScreenSize();
+    private PowerPanel powerPanel = new PowerPanel();
```

Comment:
Not wrong, but is there a reason for this to be initialized here instead of in the constructor?

Reply:
fixed